<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuadCopterWindow</class>
 <widget class="QMainWindow" name="QuadCopterWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Quadcopter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="statusBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Status</string>
    </property>
    <widget class="QGroupBox" name="componentBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>120</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Components</string>
     </property>
     <widget class="QLabel" name="cameraLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Camera</string>
      </property>
     </widget>
     <widget class="QLabel" name="gpsLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>GPS</string>
      </property>
     </widget>
     <widget class="QLabel" name="gyroscopeLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Gyroscope</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="rotorStatusBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>471</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Rotors</string>
     </property>
     <widget class="QLabel" name="flLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Front Left</string>
      </property>
     </widget>
     <widget class="QLabel" name="blLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>47</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Back Left</string>
      </property>
     </widget>
     <widget class="QLabel" name="frLabel">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Front Right</string>
      </property>
     </widget>
     <widget class="QLabel" name="brLabel">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>60</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Back Right</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="flValue">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="blValue">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="frValue">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>20</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="brValue">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>60</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QProgressBar" name="batteryBar">
     <property name="geometry">
      <rect>
       <x>237</x>
       <y>20</y>
       <width>241</width>
       <height>23</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
     <property name="invertedAppearance">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="batteryLabel">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Battery status:</string>
     </property>
    </widget>
    <widget class="QLabel" name="currentModeLabel">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>20</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Current control mode: </string>
     </property>
    </widget>
    <widget class="QLabel" name="currentModeValue">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>20</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Automatic</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="positionBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>571</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Position</string>
     </property>
     <widget class="QLabel" name="pitchLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>47</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Pitch</string>
      </property>
     </widget>
     <widget class="QLabel" name="yawLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>47</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Yaw</string>
      </property>
     </widget>
     <widget class="QLabel" name="rotationLabel">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>47</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Rotation</string>
      </property>
     </widget>
     <widget class="QLabel" name="altitudeLabel">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>47</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Altitude</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="pitchValue">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="yawValue">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="rotationValue">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="altitudeValue">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>30</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="headingLabel">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>10</y>
        <width>47</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>heading</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="headingValue">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>30</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="controlsBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>491</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>Controls</string>
    </property>
    <widget class="QPushButton" name="forwardButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Forward</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backwardButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Backward</string>
     </property>
    </widget>
    <widget class="QPushButton" name="leftButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Left</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rightButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Right</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="waypointBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>451</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Waypoint</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="timeLimitLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>47</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Time Limit</string>
      </property>
     </widget>
     <widget class="QLabel" name="gpsLocationLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>GPS Location</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="gpsXInput">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="gpsYInput">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="timeLimitInput">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="sendButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Send Next
Waypoint</string>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="secondsLabel">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>60</y>
        <width>47</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Seconds</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="abortButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>510</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>ABORT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="takeOff_LandButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>450</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Take Off</string>
    </property>
   </widget>
   <widget class="QPushButton" name="armButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>390</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Arm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shutdownButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>510</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Shutdown</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="settingsBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>260</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QLabel" name="cmLabel">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Centimeter</string>
     </property>
    </widget>
    <widget class="QLabel" name="maxAltitudeLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Max Altitude</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="maxAltitudeInput">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="toggleControlModeButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Toggle
Mode</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sendMaxAltitudeButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Send Max Altitude</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
